dockerfiles/che-cli/files-to-install/che.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/sh
```

Comment:
@l0rd Can you clarify why we need this script and https://github.com/eclipse/che/pull/1683/files#diff-358fea9821fbcf9770f2820a93c4c5ab. They are pretty similar.